mobile/library/common/config/config.cc

```diff
@@ -97,6 +97,18 @@ const char* socket_tag_config_insert = R"(
       "@type": type.googleapis.com/envoymobile.extensions.filters.http.socket_tag.SocketTag
 )";
 
+const char* persistent_dns_cache_config_insert = R"(
+- &persistent_dns_cache_config
+  config:
+    name: "envoy.key_value.platform"
+    typed_config:
+      "@type": type.googleapis.com/envoymobile.extensions.key_value.platform.PlatformKeyValueStoreConfig
+      key: dns_persistent_cache
+      save_interval:
+        seconds: 15
```

Comment:
How does this save interval work? I don't think we get so many DNS resolutions for it to significantly impact performance to persist on every resolution without batching every 15 seconds.